### PR TITLE
Usage and API Docs

### PR DESCRIPTION
**Pull request by todd**
nqkzq4c

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,36 @@ it with:
 OpenComponents.configure { |config| config.registry = 'http://some.other.host' }
 ```
 
+## Getting Rendered Components
+To request an OpenComponent with its rendered HTML, you can create and load a new
```
Comment (manfrin) [inline README.md]: passive voice -- prefer 'create and load' over 'you can ...'

File: README.md [hunk 2]
```diff
@@ -33,6 +33,36 @@ it with:
...
+
+Optionally, you can also specify parameters and the component version to request:
+```ruby
+component = OpenComponents::RenderedComponent.new(
+  'my-awesome-component', {name: 'Kate'}, '1.0.2'
+)
+```
+
+## Getting Pre-rendered Components
+If you'd like to perform the component rendering yourself, you can request a
```
Comment (manfrin) [inline README.md]: pv

File: README.md (hunk 1 above)
Comment (todd) [inline README.md]: I actually prefer using passive voice in documentation. Unless it's not consistent, of course.

**Comment by manfrin** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (Kimtaro) [inline README.md]: Is this really passive voice? Wouldn't that be `An OpenComponent with rendered HTML can be requested by creating and loading a new RenderedCompontent object`?

**Pull request merged**